### PR TITLE
Improve Quit check in update function

### PR DESCRIPTION
**Pull request by Thorgaran**
It's a small commit to test the PR system

### CODE REVIEW

**Comment by Descharges** [comment]
This is a pretty good PR. Thus, I will accept it inside the repository. Xoxo

File: src/Gui.rs [hunk 1]
```diff
@@ -51,14 +51,13 @@ impl Gui{
...
-            match event{
-                Event::Quit {..} => {return false;},
-                _ => {}
-            }
+        if self.events.poll_iter().any(|i| matches!(i, Event::Quit {..})) {
+            false
+        }
+        else {
+            self.canvas.present();
+            true
```
Comment (Descharges) [inline src/Gui.rs:59]: I wish there was "return" but since it's rust it's not here (tbh it's better this way but C is still fresh in my mind)

**Review by Descharges — APPROVED** [review]
Overall, pretty _**rad**_

File: src/Gui.rs (hunk 1 above)
Comment (Thorgaran) [reply src/Gui.rs:59]: Yes i agree but Rust all the way

**Pull request merged**